content/modules/_index.md

```diff
@@ -8,8 +8,11 @@ aliases: /rs/developing/modules/
 ---
 Redis Labs develops several modules that extend the core Redis feature set. Some of the features these modules provide include [querying, indexing and full-text search]({{< relref "/modules/redisearch" >}}), [JSON support]({{< relref "/modules/redisjson" >}}), and [probabalistic data structures]({{< relref "/modules/redisbloom" >}}).
 
-You can use the modules listed here with [Redis Enterprise Software]({{< relref "/rs" >}}) and [Redis Cloud]({{< relref "/rc" >}}).
+You can use these modules [Redis Enterprise Software]({{< relref "/rs" >}}).  [Redis Enterprise Cloud]({{< relref "/rc" >}}) supports a selected set of modules.  For details, see [Supported modules]({{< relref "/rc/databases/create-database#supported-modules" >}}).
```

Comment:
I think line 11 is missing the word "with" after "modules": "You can use these modules [Redis Enterprise Software]".